toolbox/certbot/cron_entry

```diff
@@ -0,0 +1 @@
+0 */12 * * * root /usr/airflow/atd-airflow/toolbox/certbot/renew.sh >> /var/log/airflow_cert_renewal.log
```

Comment:
Do you mind sticking a EOF line feed on this?

Reply:
Oh yup, thanks for catching that! 🙏